lib/teracy-dev-essential/config/rsync_recovery.rb

```diff
@@ -7,6 +7,22 @@ module Config
     class RsyncRecovery < TeracyDev::Config::Configurator
 
       def configure_common(settings, config)
+        vagrant_version = Vagrant::VERSION
+
+        if Vagrant::Util::Platform.linux?
+          if TeracyDev::Util.require_version_valid?(vagrant_version, ">= 2.2.3")
+            return unless rsync?(settings)
+
+            @cmd = 'rsync-auto'
+
+            @last_node = settings['nodes'].last['name']
+
+            return
```

Comment:
and you're removing all the existing logic, this kind of code is messy and smelly to me, it's very hard to check when the logic changes very often.

For example, `$ vagrant gatling-rsync-auto` will not auto switch to `$ vagrant up` anymore.

So please move the changes of this issue as a condition check only on the trigger stuff (configure_trigger_after function) instead.